src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java

```diff
@@ -21,7 +21,7 @@
     public final Classification classification;
     public final List<Document> documents;
     public final List<OcrDataField> ocrData;
-    public final List<String> ocrValidationWarnings;
+    public final List<String> ocrDataValidationWarnings;
```

Comment:
why? now it doesn't match the json property

Reply:
The field in the queue message is called `ocr_validation_warnings`, which is a bit against our naming convention for OCR data. I'll change the element name to `ocr_data_validation_warnings`, but that requires a change in processor and can be done separately, in steps.

Reply:
I'd suggest sticking to populating CCD exception record with validation warnings in this PR

Reply:
I don't understand what you mean

Reply:
revert this change

Reply:
I'll make this change in a previous PR. Otherwise, I'd create more inconsistencies.